src/content/developers/tutorials/how-to-write-and-deploy-an-nft.md

```diff
@@ -0,0 +1,334 @@
+    git repository: 
+    keywords: 
+    author: 
+    license: (ISC) 
+    About to write to /Users/thesuperb1/Desktop/my-nft/package.json:
+
+    {
+      "name": "my-nft",
+      "version": "1.0.0",
+      "description": "My first NFT!",
+      "main": "index.js",
+      "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+      },
+      "author": "",
+      "license": "ISC"
+    }
+
+Approve the package.json, and we’re good to go!
+
+## Step 7: Download [Hardhat](https://hardhat.org/getting-started/#overview)
+
+Hardhat is a development environment to compile, deploy, test, and debug your Ethereum software. It helps developers when building smart contracts and dApps locally before deploying to the live chain.
+
+Inside our my-nft project run:
+
+    npm install --save-dev hardhat
+
+Check out this page for more details on [installation instructions](https://hardhat.org/getting-started/#overview).
+
+## Step 8: Create Hardhat project
+
+Inside our project folder run:
+
+    npx hardhat
+
+You should then see a welcome message and option to select what you want to do. Select “create an empty hardhat.config.js”:
+
+    888    888                      888 888               888
+    888    888                      888 888               888
+    888    888                      888 888               888
+    8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
+    888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
+    888    888 .d888888 888    888  888 888  888 .d888888 888
+    888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
+    888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
+    👷 Welcome to Hardhat v2.0.11 👷‍
+    ? What do you want to do? …
+    Create a sample project
+    ❯ Create an empty hardhat.config.js
+    Quit
+
+This will generate a hardhat.config.js file for us which is where we’ll specify all of the set up for our project (on step 13).
+
+## Step 9: Add project folders
+
+To keep our project organized, we’ll create two new folders. Navigate to the root directory of your project in your command line and type:
+
+    mkdir contracts
+    mkdir scripts
+
+* contracts/ is where we’ll keep our NFT smart contract code
+
+* scripts/ is where we’ll keep scripts to deploy and interact with our smart contract
+
+## Step 10: Write our contract
+
+Now time that our environment is set up, onto more exciting stuff: *writing our smart contract code!*
+
+Open up the my-nft project in your favorite editor (we like [VSCode](https://code.visualstudio.com/)). Smart contracts are written in a language called Solidity which is what we will use to write our MyNFT.sol smart contract.‌
+
+1. Navigate to the “contracts” folder and create a new file called MyNFT.sol
+
+1. Below is our NFT smart contract code, which based off of the [OpenZepplin](https://docs.openzeppelin.com/contracts/3.x/erc721) library’s ERC721 implementation. Copy and paste the contents below into your MyNFT.sol file.
```

Comment:
`1.`  should be `2.`

Also, trying to standardize using hyphenated ERC references throughout site, ie. `ERC-721`... the mentions of ERC721 within code blocks below are fine as is